Automattic/jetpack
# Update connection ToS messaging

Our current ToS message appears in different parts of the plugin and makes it seem like we share data with a third party, which sounds more alarming than it really is.
After collaborating with the legal team (pbNhbs-auw-p2#comment-21945), we agreed on revising and improving the messaging:

- Our current ToS messaging says:
  - _"By clicking the [button name] button, you agree to our [Terms of Service](https://jetpack.com/redirect/?source=wpcom-tos) and to [share details](https://jetpack.com/redirect/?source=jetpack-support-what-data-does-jetpack-sync) with WordPress.com."_

- This PR changes the ToS messaging to read:
  - _"By clicking [button name], you agree to our [Terms of Service](https://jetpack.com/redirect/?source=wpcom-tos) and to [sync your site’s data](https://jetpack.com/redirect/?source=jetpack-support-what-data-does-jetpack-sync) with us."_





Related to: https://github.com/Automattic/jetpack-roadmap/issues/1554

## Proposed changes:


* In the connection ToS message, change "share details with WordPress.com." to "sync your sites data with us."

### Other information:

- [#] Have you written new tests for your changes, if applicable?

## Jetpack product discussion



pbNhbs-auw-p2#comment-21945

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






- Create a Jurassic Ninja site with the Beta Tester plugin running this Jetpack branch: `update/improve-tos-message`
- In the wp-admin sidebar nav, click "Jetpack".
- Just above the "Set up Jetpack" button, examine the ToS message.
Verify that it now says:
    - _"...and to [sync your site's data](https://jetpack.com/redirect/?source=jetpack-support-what-data-does-jetpack-sync) with us."_,
    instead of:
    _"...and to [share details](https://jetpack.com/redirect/?source=jetpack-support-what-data-does-jetpack-sync) with WordPress.com."_
- Now in the wp-admin sidebar nav, click "My Jetpack"
- Now in the top Welcome banner, click "Finish setting up Jetpack" button.
- Now just above the "Connect your user account" button, examine the ToS message.
Verify that it now says: _"...and to [sync your site's data](https://jetpack.com/redirect/?source=jetpack-support-what-data-does-jetpack-sync) with us."_,
- Go ahead and click the "Connect your user account" button. Then click "Approve", and then "Start for free" when presented the pricing page.
- Once connected, Go to "My Jetpack" again, Click the CTA on _each_ of the products
    - Verify that on each of the product card interstitials, there is a ToS that reads, "...and to **sync your site's data** with us.", instead of, "...and to **share details** with WordPress.com." 
    - On specifically interstitials that are bundled (i.e.- Backup and AntiSpam are bundled with Security), verify that there is now only one(1) combined ToS under both product cards. verify that both product button labels are displayed in the ToS and it reads similar to:, "By clicking Get Jetpack VaultPress Backup or Get Jetpack Security, you agree to our Terms of Service and to **sync your site's data** with us.", instead of, "...and to **share details** with WordPress.com."

**Test/check the '_Move to WordPress.com_' plugin:**

- In the Beta Tester plugin, for the "Move to WordPress.com" plugin, click the 'Manage' button and select this feature branch (`update/improve-tos-message`).
- In the wp-admin sidebar nav, click the "Move to WordPress.com" nav item. 
- Now on the "Move to WordPress.com" wp-admin page, you should see the ToS just above the "Get Started" button.  Verify the ToS now says, , "...and to **sync your site's data** with us.", instead of, "...and to **share details** with WordPress.com." 

**Test/check the '_Automattic for Agencies client_' plugin:**

- In the Beta Tester plugin, for the "Automattic for Agencies client" plugin, click the 'Manage' button and select this feature branch (`update/improve-tos-message`).
- Go to 'Plugins' page, find the "Automattic for Agencies client" plugin item, and click the "Settings" link.
- You should now see the Automattic for Agencies settings page. 
- You should see the ToS just above the "Connect this site" button.  Verify the ToS now says, , "...and to **sync your site's data** with us.", instead of, "...and to **share details** with WordPress.com." 

**Unit tests:**

- Run `jetpack test`
- For Type of project, select `js-packages`
- For which project, select `connection`
- For what tests, select `js`
- Verify that there are no failing tests.
